backend/routes.js

```diff
@@ -179,12 +179,25 @@ route.GET('/assets/{path*}', {
         return h.continue
       }
     }
+  },
+  files: {
+    relativeTo: path.resolve('dist/assets')
   }
-}, {
-  directory: {
-    path: path.resolve('./dist/assets'),
-    redirectToSlash: true
+}, function (request, h) {
+  const { subpath } = request.params
+  const basename = path.basename(subpath)
+  const extension = path.extname(subpath)
+  console.debug(`GET /assets/${subpath}`)
+  // In the build config we told our bundler to use the `-cached-[hash]` template to name immutable assets.
+  // This is useful because `dist/assets/` currently includes a few files without hash in their name.
+  if (basename.endsWith(`-cached${extension}`)) {
```

Comment:
I think `.indexOf('-cached') !== -1` would be more appropriate here, because this doesn't handle files with multiple dots near the end, for example, `/assets/js/chunk-P6QBAJFH-cached.js.map` isn't matched here, and neither would any other file that uses multiple `.`'s near the end.

Reply:
Good catch! I did think about the possibility of multiple extensions, but somehow got confused in the execution.